tools/backup/create-backup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Usage of the script:
+#
+# ./create-backup <port-number> <postgres-db-name> <postgres-username> <postgres-password> <output-dir>
+# e.g
+# ./create-backup 5432 postgres postgres your-long-postgres-password /home/user/backups
+
+set -e
+set -u
+
+PORT="$1"
+DB_NAME="$2"
+DB_USER_NAME="$3"
+DB_USER_PASSWORD="$4"
+OUTPUT_DIR="$5"
+
+TIMESTAMP="$(date +%b-%d-%y)"
+
+pg_dump --format=custom \
+	--dbname="host=localhost port=$PORT dbname=$DB_NAME user=$DB_USER_NAME password=$DB_USER_PASSWORD" > "$OUTPUT_DIR/$TIMESTAMP-backup.dump"
```

Comment:
This is going to be challenging to use in production, since we don't expose the database port.  We'll have to run it in a container that has access to the postgres instance via Docker DNS, and maybe as a cron job or something.

Reply:
So, your suggestion is to have a docker container that runs the `create-backup` script as a cron job, and outputs that file into a volume, correct?

Where should place this docker container's `Dockerfile`?

Reply:
It could live in the `docker/supabase/*` folder.  We do something like this now with how nginx re-reads its config every 6 hours, see https://github.com/Seneca-CDOT/telescope/blob/master/docker/production.yml#L152.

So this script could mount a volume like what we do with postgres in Supabase (see https://github.com/Seneca-CDOT/telescope/blob/master/docker/supabase/supabase-production.yml#L10), and dump the backup there.  Later we can move that to a remote machine, but having it at all is good for now.

Reply:
Jerry, you could file an issue on this